arwen/contexts/reservedFunctions.go

```diff
@@ -13,6 +15,7 @@ func NewReservedFunctions(scAPINames []string) *ReservedFunctions {
 
 	var empty struct{}
 	result.functionNames["claimDeveloperRewards"] = empty
```

Comment:
ohhh. Needs the map from Elrond-Go but probably in another PR.

Reply:
Yes, will do 2 PRs (Arwen plus Node).